src/yetibot/commands/github.clj

```diff
@@ -137,36 +137,32 @@
                   (f/unparse date-formatter datetime))))
       (str n " is not a number"))))
 
+(defn format-release
+  "Displays information about a release"
+  [release org-name repo]
+  (if (nil? (:status release))
+    (let [tag (:tag_name release)
+          author (get-in release [:author :login])
+          published-at (-> "YYYY-MM-dd'T'HH:mm:ssZ"
+                           (f/formatter)
+                           (f/parse (:published_at release)))]
+      (format "Release version, tagged: %s from %s/%s, was published on %s by %s" tag org-name repo
```

Comment:
For `show` commands I'd like to see more data for that release. Can we render the body or description of the release? For example, this release page contains a ton of interesting info: https://github.com/helm/helm/releases/tag/v2.11.0

Reply:
That makes sense, I'm on it ^^

Reply:
so, this is how it renders at the moment : 

![show_cmd_with_changelogs](https://user-images.githubusercontent.com/847920/47920170-b27e4f00-deb1-11e8-810d-bd030742fab2.png)

is it cool for you guys ?
thanks

Reply:
Yep, looks great!